ui/src/utils/treeUtils.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { SubscriptionTreeItemBase } from '..';
```

Comment:
The imports for this repo are weird and VS Code doesn't auto-import from the best place. I'd feel better if you fixed the `SubscriptionTreeItemBase` import to look more like the `AzExtTreeItem ` import